### PR TITLE
DOCS-272: query profiler

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

**Comment by qoega** [comment]
Diff is not only about query profiler. Can you split PR please?

**Comment by alexey-milovidov** [comment]
@qoega Usually we don't insist on splitting.

**Comment by BayoNet** [comment]
> Diff is not only about query profiler. Can you split PR please?

There are just markup edits and some minor updates I've made by comments to other pulls. Usually, I try not to do so. 

Splitting this pull can take longer than it's reasonable to spend on it.

File: docs/en/getting_started/install.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ sudo apt-get install clickhouse-client clickhouse-server
 
 You can also download and install packages manually from here: <https://repo.yandex.ru/clickhouse/deb/stable/main/>.
 
+#### Packages
+
+- `clickhouse-common-static` — Installs ClickHouse compiled binary files.
+- `clickhouse-server` — Creates symbolic link for `clickhouse-server`. Installs server configuration.
+- `clickhouse-client` — Creates symbolic link for `clickhouse-client `. Installs client configurations.
```
Comment (alexey-milovidov) [inline docs/en/getting_started/install.md]: And clickhouse-local, clickhouse-format, clickhouse-obfuscator.

Comment (alexey-milovidov) [inline docs/en/getting_started/install.md]: Extra whitespace.

File: docs/en/operations/table_engines/jdbc.md [hunk 2]
```diff
@@ -54,18 +54,12 @@ mysql> select * from test;
...
-│ int_nullable       │ Nullable(Int32)    │              │                    │
-│ float              │ Float32            │              │                    │
-│ float_nullable     │ Nullable(Float32)  │              │                    │
-└────────────────────┴────────────────────┴──────────────┴────────────────────┘
+CREATE TABLE jdbc_table(
+    `int_id` Int32,
+    `int_nullable` Nullable(Int32),
+    `float` Float32,
+    `float_nullable` Nullable(Float32)
+)ENGINE JDBC('jdbc:mysql://localhost:3306/?user=root&password=root', 'test', 'test')
```
Comment (alexey-milovidov) [inline docs/en/operations/table_engines/jdbc.md]: Missing whitespace.

**Pull request merged**